src/components/Icon/icons/MediaList.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import iconColorsMap from './colors';
 
-const ArticleList = ({ fill, ...props }) => (
+const ListAlt2 = ({ fill, ...props }) => (
```

Comment:
If you rename this to `MediaList`, the const should match the file name 😄

Reply:
ugh. Had the change but didn't save/commit it. it's not my day. 🤦‍♀️